modules/source/sftp/config/sftp.xml

```diff
@@ -21,7 +21,8 @@
 		<beans:property name="password" value="${password}" />
 		<beans:property name="privateKey" value="${privateKey}" />
 		<beans:property name="privateKeyPassphrase" value="${passPhrase}" />
-		<beans:property name="allowUnknownKeys" value="true"/>
+		<beans:property name="allowUnknownKeys" value="${allowUnknownKeys}" />
+		<beans:property name="knownHosts" value="#{'UNDEFINED'.equals('${knownHostsExpression}') ? null : ${knownHostsExpression}}" />
```

Comment:
This feels weird. Can't we use something like `value="${knownHostsExpression:#{null}}"` and have the field default value be `null`?